src/test/java/org/openRealmOfStars/player/fleet/FleetListTest2.java

```diff
@@ -0,0 +1,355 @@
+package org.openRealmOfStars.player.fleet;
+
+import static org.junit.Assert.*;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Mockito.*;
+import org.openRealmOfStars.player.fleet.Fleet;
+import org.openRealmOfStars.player.fleet.FleetList;
+
+/**
+ * 
+ * Open Realm of Stars game project Copyright (C) 2016 Tuomo Untinen

```

Comment:
* Open Realm of Stars game project
* Copyright (C) 2017  <Your name>

If you want to change your name to Copyright text.